src/st/e_blue_axe_knight.h

```diff
@@ -714,8 +716,10 @@ void EntityAxeKnightRotateAxe(void) {
     g_CurrentEntity->rotate &= 0xFFF;
 }
 
-extern EInit g_EInitAxeKnightAxe;
+static u32 init_velocity_x[] = {FIX(2), FIX(2), FIX(1)};
+static u32 init_velocity_y[] = {FIX(0), FIX(0), FIX(-4)};
```

Comment:
```suggestion
static s32 init_velocity_x[] = {FIX(2), FIX(2), FIX(1)};
static s32 init_velocity_y[] = {FIX(0), FIX(0), FIX(-4)};
```
since `FIX(-4)` is negative